t/cli_options.t

```diff
@@ -109,4 +109,18 @@ make_path("$TEST_DIR/".'$HOME');
 is($options->{target}, "$TEST_DIR/".'$HOME', 'no expansion');
 remove_dir("$TEST_DIR/".'$HOME');
 
+#
+# Check if verbose on simulate invoked option
```

Comment:
Great!  Would be worth having an extra test to make sure that if `--simulate -v3` is specified then `$options->{verbose}` gets set to `3`.

Reply:
My English is in progress (WIP), but I did the best I got. The review would be cool ...